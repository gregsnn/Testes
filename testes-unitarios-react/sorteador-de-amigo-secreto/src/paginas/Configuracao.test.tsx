import { render } from "@testing-library/react";
import { RecoilRoot } from "recoil";
import Configuracao from "./Configuracao";

const mockNavigate = jest.fn()
jest.mock('react-router-dom', () => ({
  useNavigate: () => mockNavigate
}))

describe('Pagina Configuracao', () => {
  it('Deve renderizar a pagina', () => {
    const { container } = render(<RecoilRoot>
      <Configuracao />
    </RecoilRoot>)

    expect(container).toMatchSnapshot();
  });
});