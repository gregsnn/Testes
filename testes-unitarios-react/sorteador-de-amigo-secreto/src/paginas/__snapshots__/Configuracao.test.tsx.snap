// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagina Configuracao Deve renderizar a pagina 1`] = `
<div>
  <form>
    <input
      placeholder="Insira os nomes dos participantes"
      type="text"
      value=""
    />
    <button
      disabled=""
    >
      Adicionar
    </button>
    
  </form>
  <ul />
  <footer
    class="text-center"
  >
    <button
      disabled=""
      type="button"
    >
      Iniciar brincadeira
    </button>
  </footer>
</div>
`;
